src/com/lftechnology/java/training/alina/basic/palindrome/LargestPalindrome.java

```diff
@@ -4,58 +4,53 @@
 import java.util.logging.Logger;
 
 /**
- * A palindromic number reads the same both ways. The largest palindrome made
- * from the product of two 2-digit numbers is 9009 = 91 × 99. Find the largest
- * palindrome made from the product of two 3-digit numbers.
+ * A palindromic number reads the same both ways. The largest palindrome made from the product of two 2-digit numbers is 9009 = 91 × 99.
+ * Find the largest palindrome made from the product of two 3-digit numbers.
  * 
  * @author Alina Shakya <alinashakya@lftechnology.com>
  *
  */
 public class LargestPalindrome {
 
-	private static final Logger LOGGER = Logger
-			.getLogger(LargestPalindrome.class.getName());
+    private static final Logger LOGGER = Logger.getLogger(LargestPalindrome.class.getName());
 
-	public static void main(String[] args) {
-		try {
-			getLargestPalindrome();
-		} catch (Exception e) {
-			LOGGER.log(Level.WARNING, "Exception Message : {0}",
-					new Object[] { e.getMessage() });
-		}
+    public static void main(String[] args) {
+        try {
+            getLargestPalindrome();
+        } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)